release/prepareRelease.sh.cmake

```diff
@@ -91,7 +91,7 @@ extract_debug_symbols()
 	objcopyTarget=""
 	case "$SYSTEM_NAME" in
 		Windows)
-			objcopyTarget="-pc-mingw32"
+			objcopyTarget="-w64-mingw32"
```

Comment:
Couldn't we get this from the compiler? Needs double checking but that is how CMake works: Find the tools from the compiler prefix (not the variable but the actual exe). We can put the compiler in here (this is configured) and match it with simple regex. This would replace at least lines 74-129

Reply:
yep perhaps simply we let it "find" via cmake and put the objcopy as a variable in there?

Reply:
Maybe it's even easier to just translate this into cmake and run it in script mode? It would keep all the logic here and makes maintenance easier. E.g. I'm much better in cmake than shell